### PR TITLE
Examples as tutorials

### PR DESCRIPTION
**Pull request by v1nvn**

### CODE REVIEW

File: models/examples/02_dto_examples.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        output.log("DTOs are classes with typed properties which will serialize and deserialize to and from JSON.")
+
+        class Token(DTO):
+            symbol: str
+            total_supply: int = 1000
+
+        output.log("""
+class Token(DTO):
+    symbol: str
+    total_supply: int = 1000""")
```
Comment (newtectonics) [inline models/examples/02_dto_examples.py]: I feel like putting in a fake total_supply will be confusing

File: models/examples/06_token_examples.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
+
+from credmark.cmf.model import Model
+from credmark.cmf.model.errors import ModelRunError
+from credmark.cmf.types import Address, NativeToken, Token
+from credmark.dto import DTO, DTOField
+from models.dtos.example import ExampleModelOutput
+
+
+class _TokenLoadingInput(DTO):
```
Comment (newtectonics) [inline models/examples/06_token_examples.py]: why the _ before the name?

File: models/examples/06_token_examples.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            raise ModelRunError(
+                message="Token cannot be initialized with an invalid address, "
+                "an exception was NOT caught, and the example has FAILED")
+        except ValueError as _e:
+            output.log_error(_e)
+            output.log_error(
+                "Attempting to initialize a Token with invalid address "
+                "raises ValueError")
+
+        # try:
```
Comment (newtectonics) [inline models/examples/06_token_examples.py]: let's take out the comments before we publish.

**Review by newtectonics — COMMENTED** [review]
<3 this. a few questions and then we can publish :)

File: models/examples/e_05_contract.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+class ExampleAddress(Model):
+    def run(self, _) -> ExampleModelOutput:
+        output = ExampleModelOutput(
+            github_url="https://github.com/credmark/credmark-models-py/blob/main/models/examples/e_05_contract.py",
+            documentation_url="https://developer-docs.credmark.com/en/latest/reference/credmark.cmf.types.contract.Contract.html")
+
+        output.log("This model demonstrates the functionality of the Contract class")
+
+        output.log("Contract is a subclass of Account, and is initialized with an address.")
+        contract = Contract(address=CMK_ADDRESS)
```
Comment (newtectonics) [inline models/examples/e_05_contract.py:25]: Can we use a contract that's NOT a token, to introduce these concepts one at a time? We could use a CMK Vesting contract or an aave reserve or something

File: models/examples/06_token_examples.py (hunk 2 above)
Comment (v1nvn) [reply models/examples/06_token_examples.py]: leading _ hints that the class is internal.

File: models/examples/02_dto_examples.py (hunk 1 above)
Comment (v1nvn) [reply models/examples/02_dto_examples.py]: Wanted to a demonstrate a field with default value. I'll replace it with `decimals: int = 18`

File: models/examples/06_token_examples.py (hunk 3 above)
Comment (v1nvn) [reply models/examples/06_token_examples.py]: OK

File: models/examples/e_05_contract.py (hunk 4 above)
Comment (v1nvn) [reply models/examples/e_05_contract.py:25]: Sure.

File: models/examples/06_token_examples.py (hunk 2 above)
Comment (newtectonics) [reply models/examples/06_token_examples.py]: Interesting, ok

**Pull request merged**